goldenmaster/examples/mqttqml/main.cpp

```diff
@@ -102,24 +102,31 @@ int main(int argc, char *argv[]){
     // Prepare Factory before app is created.
     ApiGear::Mqtt::Client client("UniqueClientForQmlExample");
     ApiGear::Mqtt::ServiceAdapter service("UniqueService");
+    

```

Comment:
this new line change looks kind of accidental.
If it's intentionally please put it into separate "style" commit